bundle/compliance/cis_gcp/rules/cis_4_8/rule.rego

```diff
@@ -0,0 +1,24 @@
+package compliance.cis_gcp.rules.cis_4_8
+
+import data.compliance.lib.common
+import data.compliance.policy.gcp.data_adapter
+
+default is_shielded_vm = false
```

Comment:
any reason to use defailt ... = false instead of else = false clause in the audit?

Reply:
no reason, converted.

Reply:
please convert the other rules too